bin/make-consensus.py

```diff
@@ -181,13 +200,25 @@ def main():
         sample = result.stdout.strip()
 
     consensusFile = join(tempdir, 'consensus.fasta')
-    result = e.execute(
-        "bcftools consensus --sample '%s' --iupac-codes %s --fasta-ref "
-        "'%s' '%s' > '%s'" %
-        (sample, maskArg, args.reference, vcfFile, consensusFile))
 
-    consensus = list(FastaReads(consensusFile))[0]
+    if args.ivar:
+        ivarConsensusFile = join(tempdir, 'temporary-consensus')
+        result = e.execute(
+            "samtools mpileup -A -Q 0 %s | "
+            "ivar consensus -p %s -q 20 -t %s -m %s" % (
+                args.bam, ivarConsensusFile, args.ivarFrequencyThreshold,
+                args.maskLowCoverage))
+
+        result = e.execute(
+            "mv %s %s" % (ivarConsensusFile + '.fa', consensusFile))
 
+    else:
+        result = e.execute(
+            "bcftools consensus --sample '%s' --iupac-codes %s --fasta-ref "
+            "'%s' '%s' > '%s'" %
+            (sample, maskArg, args.reference, vcfFile, consensusFile))
```

Comment:
Just use `%r` everywhere.   In an `f-string` use `{variable!r}`.

Reply:
Done.